tools/genevergreen/main.go

```diff
@@ -97,13 +106,66 @@ func generateRepoTasks(toolName string) *shrub.Configuration {
 
 		c.Variants = append(c.Variants, v)
 	}
+}
+
+func generatePostPkgTasks(c *shrub.Configuration, toolName string) {
+	v := &shrub.Variant{
+		BuildName:        fmt.Sprintf("pkg_smoke_tests_docker_%v_generated", toolName),
+		BuildDisplayName: fmt.Sprintf("Generated post packaging smoke tests (Docker / %v)", toolName),
+		DistroRunOn:      []string{"ubuntu1804-small"},
+	}
+
+	for _, os := range oses {
+		t := &shrub.Task{
+			Name: fmt.Sprintf("pkg_test_%v_docker_%v", toolName, os),
+		}
+		t = t.Dependency(shrub.TaskDependency{
+			Name:    "package_goreleaser",
+			Variant: fmt.Sprintf("goreleaser_%v_snapshot", toolName),
+		}).Function("clone").FunctionWithVars("docker build", map[string]string{
+			"tool_name": toolName,
+			"image":     postPkgImg[os],
+		})
+		c.Tasks = append(c.Tasks, t)
+		v.AddTasks(t.Name)
+	}
+
+	c.Variants = append(c.Variants, v)
+}
+
+func generatePostPkgMetaTasks(c *shrub.Configuration, toolName string) {
+	if toolName != atlascli {
+		return
+	}
+
+	v := &shrub.Variant{
+		BuildName:        fmt.Sprintf("pkg_smoke_tests_docker_meta_%v_generated", toolName),
+		BuildDisplayName: fmt.Sprintf("Generated post packaging smoke tests (Meta / %v)", toolName),
+		DistroRunOn:      []string{"ubuntu1804-small"},
```

Comment:
[non-blocking] can we make `ubuntu1804-small` a const and make sure we use it everywhere so when we need to update it is in a single place?
something like
```go
const runOn = "ubuntu1804-small"
```